src/wyzeapy/services/camera_service.py

```diff
@@ -110,7 +110,3 @@ async def turn_off_notifications(self, camera: Camera):
         ]
 
         await self._set_property_list(camera, plist)
-
-    def register_updater(self, camera: Camera, interval):
-        dev_updater = DeviceUpdater(self, camera, interval)
-        BaseService._update_manager.add_updater(dev_updater)
```

Comment:
I don't think this is being used anymore. The camera and switch platforms forward this to the method in `base_service`

Reply:
Have you tested this against switches and cameras?

Reply:
Against cameras yes, I don't have any switches

Reply:
Ok cool so I will publish as a prerelease so we can get some testing before stable